dpytools/config/properties/base.py

```diff
@@ -4,17 +4,16 @@
 
 @dataclass
 class BaseProperty(metaclass=ABCMeta):
-    name: str
-    value: Any
+    _name: str
+    _value: Any
 
-    # TODO: getter
-    # allow someone to get the property
+    @property
```

Comment:
the get and set pattern is correct 👍 , but can we do it for _value as. well please, we _really_ dont want people chaging that.